src/stflpp.cpp

```diff
@@ -16,7 +17,7 @@ namespace newsbeuter {
  */
 
 stfl::form::form(const std::string& text) : f(0) {
-	ipool = stfl_ipool_create((std::string(nl_langinfo(CODESET)) + "//TRANSLIT").c_str());
+	ipool = stfl_ipool_create(utils::translit(std::string(nl_langinfo(CODESET)), "WCHAR_T").c_str());
```

Comment:
We can encapsulate some string conversion if we add `std::string translit(const char* tocode, const std::string& fromcode)`.

(There's also final `c_str()` call, but I suspect this is more tricky due to time of life details. I'll deal with this myself later.)